cmd/models/import.go

```diff
@@ -46,6 +47,15 @@ func NewImportCmd() *cobra.Command {
 				return err
 			}
 
+			workspaceConfig, err := workspace.GetWorkspaceConfig()
+			if err != nil {
+				return err
+			}
+
+			if len(projectId) == 0 {
+				projectId = workspaceConfig.ProjectId
+			}
+
 			if len(projectId) == 0 {
 				projects, err := project.GetProjects(ctx)
```

Comment:
add option to select project if you don't have workspace and project id

Reply:
This is what happening, double check me